### ISSUE TITLE
[Feature Req] Multiple Playlist Support (like Foobar 2000)

### ISSUE THREAD

**Issue by ShikiSuen** [iina/iina#2826]
**What you want IINA to do:**

Adding support of multiple playlist (so user can perform hot-switch among them), similar to what Foobar 2000 does on Windows and macOS.

**What IINA does currently:**

It has only one playlist in the app.

**Examples of other projects that have something similar:**
![image](https://user-images.githubusercontent.com/3164826/76999602-3bf4cb00-699a-11ea-92b7-078de5061b78.png)

**Comment by heavyk** [comment]
thanks for the foobar 2000 suggestion! it's exactly what I'm looking for. the iina playlist in music mode is very small

👍

**Comment by ShikiSuen** [comment]
@heavyk This issue complaint won't work unless being signed and addressed by the devs.

**Comment by heavyk** [comment]
yeah, after looking through the issues (and I found this one), I realised that there was no hope for getting a change, so I just looked for an alternative and foobar2000 is working really well for me now

**Comment by ShikiSuen** [comment]
@heavyk They may either have no clue how to change the design of their UI to adopt this feature request, nor these current devs are busy doing their own matters (which is obviously more important than this voluntary project).

**Comment by speckofthecosmos** [comment]
Spent all day trying to find a replacement for kmplayer on the Mac. So far IINA is the closest. Looking for the ability to delete a file and switch playlists as I work through folders of videos to sort and prune. Just a vote for this feature :)
